db/migrate/20160827111345_add_latitude_and_longitude_to_drops.rb

```diff
@@ -0,0 +1,21 @@
+class AddLatitudeAndLongitudeToDrops < ActiveRecord::Migration
+
+  class Drop < ActiveRecord::Base
+    belongs_to :place
+  end
+
+  def up
+    add_column :drops, :latitude, :float
+    add_column :drops, :longitude, :float
+
+    Drop.reset_column_information
```

Comment:
ActiveRecord checks the database to create the attributes for its models. I'm modifying the underlying database table for `Drop` and I want to continue working with `Drop` after altering its schema. `reset_column_information` basically tells it to reload its schema info from the database. That way I'm making sure I have the two new columns (lat + lng) available in following lines.